RadixWallet.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -306,6 +306,15 @@
         "version" : "0.5.0"
       }
     },
+    {
+      "identity" : "swift-syntax",
```

Comment:
does this increase build time? if it does, we ought to bump timeout from 18 to.. 20 min?